tbconnect/views.py

```diff
@@ -21,10 +21,24 @@ def perform_create(self, serializer):
 
         profile = HealthCheckUserProfile.objects.get_or_prefill(msisdn=instance.msisdn)
         profile.update_from_tbcheck(instance)
+        profile.update_tbconnect_group_arm()
         profile.save()
 
         return instance
 
+    def perform_update(self, serializer):
+        """
+        To update commit_get_tested to an existing TBCheck
+        """
+        instance = serializer.save()
+
+        if instance.commit_get_tested:
+            tbcheck = TBCheck.objects.filter(msisdn=instance.msisdn).last()
+            tbcheck.commit_get_tested = instance.commit_get_tested
+            tbcheck.save()
+
+        return instance
+
```

Comment:
Do we need this function here? we only overwrite perform_create because we want to edit the profile with the latest details